runtime-sdk/modules/contracts/src/test.rs

```diff
@@ -513,7 +513,9 @@ impl Runtime for ContractRuntime {
 }
 
 #[test]
-fn test_hello_contract_subcalls() {
+fn test_hello_contract_subcalls_overflow() {
```

Comment:
:information_source: apparently this was always a stack overflow test. this renames it to something more descriptive. in the same change, we're adding a new non-stack-overflow test with the old name